src/gpu/gpu_shuffle_list.cpp

```diff
@@ -16,15 +16,13 @@
 
 #include "gpu/gpu_impl_list.hpp"
 
+#include "gpu/generic/sycl/ref_shuffle.hpp"
```

Comment:
Since we should not enable these for Intel vendor it should go under a condition.

Reply:
Done